eslint.config.js

```diff
@@ -17,7 +17,7 @@ export default [
 	includeIgnoreFile(gitignorePath),
 	...compat.extends('brightspace/lit-config').map(c => ({
 		...c,
-		files
+		files: c.files ?? files
```

Comment:
Overrides set for specific files (e.g. demos or lang files) where being set for all files.